server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -246,15 +246,16 @@ private DivTag renderBlockList(
       ImmutableList<BlockDefinition> blockDefinitions,
       long focusedBlockId,
       int level) {
-    DivTag container = div().withClass("pl-" + level * 2);
+    DivTag container = div();
     String genericBlockDivId = "block_list_item_";
     for (BlockDefinition blockDefinition : blockDefinitions) {
 
       // TODO: Not i18n safe.
       int numQuestions = blockDefinition.getQuestionCount();
       String questionCountText = String.format("Question count: %d", numQuestions);
       String blockName = blockDefinition.name();
-
+      // indendation value for enums and repeaters
+      int listIndendationFactor = 4 + (level * 2);
```

Comment:
(blocking) I'd still like to see the 4 and the 2 put in named constants. I don't fully understand what each value is. 

Something like BASE_INDENTATION_SIZE = 4. I'm not sure I understand what the 2 should be called.

Reply:
TBH- the previous code calculated like this-   DivTag container = div().withClass("pl-" + level * 2); but the pl value was overridden after the tailwind update. I carried over similar calculation, but for px. may be call it "FACTOR_INCREASE_ON_LEVEL"?